go.mod

```diff
@@ -4,28 +4,28 @@ go 1.15
 
 replace (
 	// Bazel is seemingly broken with newer versions of this package
-	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
```

Comment:
Nit: Given that we don't use bazel here, we can probably remove that comment :)

/lgtm
/approve
/hold

If you want to fix the nit, otherwise feel free to hold cancel

Reply:
@alvaroaleman comment removed :)
l